components/ReplyFeedback.js

```diff
@@ -1,22 +1,14 @@
-import React, { PureComponent } from 'react';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { feedbackStyle } from './ReplyFeedback.styles';
 
-export default class ReplyFeedback extends PureComponent {
+class ReplyFeedback extends Component {
```

Comment:
Why instead `PureComponent` by normal `Component`?

Reply:
Because we only export `connect()(ReplyFeedback)`, which already does pure rendering by default

Reply:
Great, I get it!